orchestra/cmds/shell.py

```diff
@@ -41,12 +41,15 @@ def handle_shell(args):
         else:
             cd_to = os.getcwd()
 
+    if command:
+        p = run_script(" ".join(command), environment=env, strict_flags=False, check_returncode=False, cwd=cd_to)
```

Comment:
You're turning an `argv` into a space separated string for the shell to parse. This will turn `['asd', '"', '"']` into `asd ""` which is not what we want. Use [`shlex.join`](https://docs.python.org/3/library/shlex.html#shlex.join), which is available in Python 3.8, therefore [we need to backport it](https://github.com/python/cpython/blob/3.9/Lib/shlex.py#L318).

Reply:
Good point, I fixed it as you suggested